src/operator/tensor/matrix_op-inl.h

```diff
@@ -675,20 +675,31 @@ inline void GetIndexRange(const mxnet::TShape& dshape,
         b = len - 1;
       }
       CHECK_LT(b, len) << "slicing with begin[" << i << "]="
-                       << b << " exceends limit of " << len;
+                       << b << " exceeds limit of " << len;
```

Comment:
Can we give a more informative error message than just `len`? e.g. "the shape size `len` in dimension `i`"?

Reply:
This error message is in accordance with the other error messages we have in this function. I am just correcting the typo here. Do you think we should change all the error messages?

Reply:
Yes, I think providing a clear error message will improve user experience significantly.

Reply:
Modified the error message to be more informative.